src/components/RefinementList/RefinementList.js

```diff
@@ -59,16 +59,17 @@ class RefinementList extends Component {
 
     return (
       <RefinementListItem
+        templateKey="item"
+        key={key}
         facetValueToRefine={facetValue.value}
-        handleClick={this.handleItemClick}
+        handleClick={facetValue.count ? this.handleItemClick : () => {}}
```

Comment:
This change breaks the RefinementList widget - we can't unselect an item that has a count of zero.

Reply:
Agreed to remove support for disabled items for now.